org.eclipse.xtext/src/org/eclipse/xtext/service/OperationCanceledManager.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2014, 2016 itemis AG (http://www.itemis.eu) and others.
```

Comment:
Shouldn't the dates on this be updated as well?

Reply:
Correct. I'll update this for this change set and consider it for more changes to come.